make_release/Readme.md

```diff
@@ -25,40 +25,49 @@
 
 ## 1. Minor bump of the version ([example][nushell bump example])
 - [ ] in the repo of Nushell, run `/path/to/nu_scripts/make_release/bump-version.nu`
-- [ ] Also commit `Cargo.lock` AFTER running a Cargo command like `cargo check --workspace`
+- [ ] then, ensure there are no compilation errors with any combination of features by running:
+    ```nushell
+    cargo hack --all --feature-powerset --skip default-no-clipboard,stable,wasi,mimalloc check
+    ```
+    (this will take a while...)
+- [ ] check for build errors by running:
+    ```nushell
+    cargo hack --all build
+    ```
+    (this will build each crate with default features)
```

Comment:
In my eye that is a slightly different step than the mechanical version bumping (with a `cargo check` to not miss it in the `Cargo.lock`) but should do the trick.